client/state/memberships/reducer.js

```diff
@@ -3,13 +3,19 @@
 /**
  * Internal dependencies
  */
+import config from 'config';
 import { combineReducers } from 'state/utils';
 import connectedAccounts from './connected-accounts/reducer';
 import productList from './product-list/reducer';
 import subscriptions from './subscriptions/reducer';
 
-export default combineReducers( {
-	connectedAccounts,
-	productList,
+const reducers = {
```

Comment:
Should this be `let reducers = {` since the reducers var is updated later on?

Reply:
Objects and arrays are not simple objects. What is defined by `const reducers` is actually a reference to the object that can be later modified.
What we cannot do is to override reducers with something else.
But if we dont change the reference, but only a property of the object that it points to - we can.

Reply:
TIL Thanks! 👍